packages/dfw/src/dfwcomponents/WorkOrderCell/WorkOrderCell.tsx

```diff
@@ -2,40 +2,43 @@ import {
     Button,
     Cell,
     EDSStyleSheet,
-    Label,
     Typography,
     useBreakpoint,
     useStyles,
+    useToken,
 } from "@equinor/mad-components";
-import moment from "moment";
 import React, { useMemo } from "react";
 import { View } from "react-native";
+import { PropertyRow } from "../PropertyRow";
 import { PropertyList } from "./PropertyList";
 import { StatusIcon } from "./StatusIcon";
 import { WorkOrderCellProps } from "./types";
 import { getStatusIconsAndLabels } from "./utils";
 
 export const WorkOrderCell = ({
     title,
+    workOrderId,
+    workOrderType,
     maintenanceType,
-    showSymbols,
-    symbolDirection = "column",
+    showSymbols = true,
     valueColor = "textTertiary",
     isHseCritical,
     isProductionCritical,
-    overwriteLabel,
     style,
     startJobButton,
     readyForOperationButton,
     tecoButton,
     ...rest
 }: WorkOrderCellProps) => {
     const breakpoint = useBreakpoint();
-    const styles = useStyles(themeStyles, { symbolDirection });
+    const token = useToken();
+    const styles = useStyles(themeStyles);
 
     const anyButtonVisible =
         startJobButton?.visible ?? readyForOperationButton?.visible ?? tecoButton?.visible;
-    const currentDate = moment();
+
+    const currentDate = useMemo(() => new Date(), []);
```

Comment:
Just a question of curiosity, not related to your code changes that much.
Why is this current date being passed in as a prop to the `PropertyList` component? Should it not be able to get this date itself?